bluesky_queueserver/manager/start_manager.py

```diff
@@ -181,6 +184,22 @@ def start_manager():
         config['kafka']['topic'] = args.kafka_topic
         config['kafka']['bootstrap'] = args.kafka_server
 
+    if args.profile_collection_path:
+        pc_path = args.profile_collection_path
+        pc_path = os.path.abspath(os.path.expanduser(pc_path))
+        if not os.path.exists(pc_path):
```

Comment:
I'd better use `os.path.isdir` here.

Reply:
I guess I should put two check here as well, because this is where the consistency of command-line parameters should be checked (if possible).